internal/server/server.go

```diff
@@ -129,6 +114,63 @@ func New(conf config.Config) (*Server, error) {
 		}
 	}
 
+	go func() {
+		log := slog.With("tunnel_groups_path", conf.TunnelGroupsPath)
+		defer log.Info("Closing tunnel groups watcher")
+
+		for groups := range groupChan {
+			log.Debug("Tunnel groups configuration update recieved")
```

Comment:
```suggestion
			log.Debug("Tunnel groups configuration update received")
```

i before e except after c ;)